internal/operands/common-templates/reconcile_test.go

```diff
@@ -258,6 +277,63 @@ var _ = Describe("Common-Templates operand", func() {
 			}
 		})
 	})
+
+	Context("total_restored_common_templates metric", func() {
+		var template *templatev1.Template
+		var initialMetricValue float64
+
+		BeforeEach(func() {
+			_, err := operand.Reconcile(&request)
+			Expect(err).ToNot(HaveOccurred())
+
+			template = &testTemplates[0]
+			template.Namespace = namespace
+
+			desc, value := getCommonTemplatesRestoredMetric()
+			Expect(desc).To(ContainSubstring("total_restored_common_templates"))
+			initialMetricValue = value
+		})
+
+		It("should increase by 1 when one template is restored", func() {
+			cloneTemplate := template.DeepCopy()
+			escapedTemplateTypeLabel := strings.ReplaceAll(TemplateTypeLabel, "/", "~1")
```

Comment:
Using `Patch()` is complex for a unit test. Can you instead call `Get()` followed by `Update()`?

Reply:
updated